metricbeat/_meta/fields.common.yml

```diff
@@ -38,3 +38,13 @@
       example: elasticsearch
       description: >
         Name of the service metricbeat fetches the data from.
+
+    - name: event.duration
+      type: long
+      description: >
+        Duration of the event in nano seconds.
+
+    - name: event.dataset
```

Comment:
`event.duration` and `event.dataset` already exist in `libbeat/_meta/fields.ecs.yml` - do we need to add them here as well?

Reply:
Ah, nevermind, this is going into 6.x.